libs/platform/src/lib/table/components/table-toolbar/table-toolbar.component.ts

```diff
@@ -52,6 +52,9 @@ export class TableToolbarComponent implements TableToolbarWithTemplate, AfterVie
     @Input()
     hideSearchInput = false;
 
+    /** Toggle to expand and collapse all feature */
```

Comment:
```suggestion
    /** Whether or not to show "expand all" and "collapse all" buttons */
```
I think for description, this fits better